experimental/LieAlgebras/src/root_systems.jl

```diff
@@ -0,0 +1,164 @@
+mutable struct RootSystem <: AbstractVector{fmpq}
+  roots::Vector{Vector{Int}}
+  simple_roots::Vector{Vector{Int}}
+  positive_roots::Vector{Vector{Int}}
+  root_system_type::Vector{Union{String, Int64}}
```

Comment:
How about a tuple instead?
```suggestion
  root_system_type::Tuple{String, Int64}
```